File: Python/coin.py
import random
import os

i = 0
outcomes = []
flips = input("How many times do you want to flip a coin? ")
files = os.listdir("C:\\Users\\PC\\Desktop\\Python\\Coin flip\\Data")
while(i<int(flips)):
    outcomes.append(random.randint(0,1))
    i+= 1
with open (f"C:\\Users\\PC\\Desktop\\Python\\Coin flip\\Data\\Flips{len(files)+1}.txt", "x") as x:
    for number in outcomes:
        if number == 0:
            x.write("Heads")
            x.write("\n")
        else:
            x.write("Tails")
            x.write("\n")

File: C/coin.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(){
	srand(time(NULL));

	int i = 0, flips = 0, randint;
	printf("How many times do you want to flip a coin? ");
	scanf("%d", &flips);
	FILE *data = fopen("data.txt", "w");

	while(i<flips){
		randint = rand() % 2;
		fprintf(data, "%d\n", randint);
        i++;
	}

	fclose(data);

	return 0;
}
